modules/rosa-sts-overview-of-the-default-cluster-specifications.adoc

```diff
@@ -44,7 +44,7 @@ endif::rosa-standalone[]
 ifndef::rosa-hcp[]
 * Default AWS region for installations using the {cluster-manager-first} {hybrid-console-second}: us-east-1 (US East, North Virginia)
 endif::rosa-hcp[]
-* Default AWS region for installations using the `rosa` CLI: Defined by your `aws` CLI configuration
+* Default AWS region for installations using the {product-title} (ROSA) CLI (`rosa`): Defined by your `aws` CLI configuration
```

Comment:
product title is expanded above so this can just be ROSA CLI

Reply:
Maybe, "ROSA CLI (`rosa`)" here?